README.md

```diff
@@ -94,11 +111,22 @@ You can use `--help` flag to see all arguments.
 | `--folder` | String. Pointing to `./validator_keys` by default | The folder path for the keystore(s) and deposit(s) |
 | `--chain` | String. `mainnet` by default | The chain setting for the signing domain. |
 
+###### `existing-mnemonic` Arguments
+
+You can use `existing-mnemonic --help` to see all arguments. Note that if there are missing arguments that the CLI needs, it will ask you for them.
+
+| Argument | Type | Description |
+| -------- | -------- | -------- |
+| `--validator_start_index` | Non-negative integer | The index of the first validator's keys you wish to generate. If this is your first time generating keys with this mnemonic, use 0. If you have generated keys using this mnemonic before, use the next index from which you want to start generating keys from (eg, if you've generated 4 keys before (keys #0, #1, #2, #3), then enter 4 here.|
+| `--num_validators`  | Non-negative integer | The number of signing keys you want to generate. Note that the child key(s) are generated via the same master key. |
+| `--folder` | String. Pointing to `./validator_keys` by default | The folder path for the keystore(s) and deposit(s) |
+| `--chain` | String. `mainnet` by default | The chain setting for the signing domain. |
```

Comment:
It looks good to add the descriptions of other arguments to`"@click.option(..., help="<description>")` as well (currently, only the `--mnemonic-password` has it)

Reply:
https://github.com/ethereum/eth2.0-deposit-cli/pull/123/commits/d3149c688a30f4af33ff0344730757dbda944f1a Added help parameters for all the commands and arguments.